keyboards/hineybush/physix/readme.md

```diff
@@ -0,0 +1,15 @@
+# physix
+
+![PhysiX](https://geekhack.org/index.php?topic=103049.0)
```

Comment:
This needs to be an actual image.

Reply:
PCBs and cases in production, will update once they're available